src/components/DocExplorer.js

```diff
@@ -496,7 +496,7 @@ class TypeDoc extends React.Component {
                 {': '}
                 <TypeLink type={field.type} onClick={onClickType} />
                 {
-                  field.isDeprecated &&
+                  field.deprecationReason &&
```

Comment:
>  is it possible to show that inline somehow?

It'll probably be something like:
```js
{
  field.deprecationReason && [
    <span className="doc-alert-text">{' (DEPRECATED)'}</span>,
    <MarkdownContent
      className="doc-alert-text"
      markdown={field.deprecationReason}
    />
  ]
}
```
Instead of reusing `doc-alert-text`, though, it'd be nice to get a grey-ish text for the deprecation reason.